pkg/network/domainspec/generators_test.go

```diff
@@ -206,7 +236,75 @@ var _ = Describe("Pod Network", func() {
 				}
 				specGenerator = NewPasstLibvirtSpecGenerator(
 					passtIface, nil, istioVmi)
-				Expect(getPorts(specGenerator)).To(Equal("-t ~15000,~15001,~15004,~15006,~15008,~15009,~15020,~15021,~15053,~15090 -u all"))
+
+				expectedPortFwd := []api.InterfacePortForward{
+					{
+						Proto: "tcp",
+						Ranges: []api.InterfacePortForwardRange{
+							{Start: 15000, Exclude: "yes"}, {Start: 15001, Exclude: "yes"},
+							{Start: 15004, Exclude: "yes"}, {Start: 15006, Exclude: "yes"},
+							{Start: 15008, Exclude: "yes"}, {Start: 15009, Exclude: "yes"},
+							{Start: 15020, Exclude: "yes"}, {Start: 15021, Exclude: "yes"},
+							{Start: 15053, Exclude: "yes"}, {Start: 15090, Exclude: "yes"},
+						},
+					},
+				}
+
+				Expect(specGenerator.generatePortForward()).To(Equal(expectedPortFwd))
+			})
+
+			It("should set passt domain interface", func() {
+				istioVmi := api2.NewMinimalVMI("test")
+				istioVmi.Annotations = map[string]string{istio.ISTIO_INJECT_ANNOTATION: "true"}
+
+				testDom := api.NewMinimalDomain("test")
+				testAlias := api.NewUserDefinedAlias("default")
+				testModel := &api.Model{Type: "virtio"}
+				testDomIface := api.Interface{Alias: testAlias, Model: testModel}
+				testDom.Spec.Devices.Interfaces = append(testDom.Spec.Devices.Interfaces, testDomIface)
+
+				vmiSpecIface := &v1.Interface{
+					Name:                   "default",
+					MacAddress:             "02:02:02:02:02:02",
+					InterfaceBindingMethod: v1.InterfaceBindingMethod{Passt: &v1.InterfacePasst{}},
+					Ports: []v1.Port{
+						{Protocol: "udp", Port: 100}, {Protocol: "udp", Port: 200},
+						{Protocol: "tcp", Port: 8080},
+						{Port: 80},
+					},
+				}
+
+				specGenerator = NewPasstLibvirtSpecGenerator(vmiSpecIface, testDom, istioVmi)
+
+				expectedIface := &api.Interface{
+					Type:    "user",
+					Backend: &api.InterfaceBackend{Type: "passt", LogFile: passtLogFile},
+					Source:  api.InterfaceSource{Device: "eth0"},
+					Alias:   testAlias,
+					Model:   testModel,
+					MAC:     &api.MAC{MAC: "02:02:02:02:02:02"},
+					PortForward: []api.InterfacePortForward{
+						{
+							Proto: "tcp",
+							Ranges: []api.InterfacePortForwardRange{
+								{Start: 15000, Exclude: "yes"}, {Start: 15001, Exclude: "yes"},
```

Comment:
It cannot be const because its array, exporting it to var in the same tests Context for reuse is not recommened because it can be mutated.
I can export it to func that return this array, WDYT?